Src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveData/ExpectedOutputs/SocialSecurityNumbers.sarif

```diff
@@ -23,9 +23,6 @@
                   "text": "'{0}' contains an apparent {1}."
                 }
               },
-              "defaultConfiguration": {
```

Comment:
>defaultConfiguration [](start = 15, length = 20)

We have an issue with specifying a default configuration currently. If you do this, you can't remap a result level to 'Warning' because that's a default value which is stripped at serialization time. Let's talk about this later!